.cspell.json

```diff
@@ -835,6 +835,7 @@
     "Kinect",
     "kinshicho",
     "kitti",
+    "kitware",
```

Comment:
In this case, please let me use the capitalized one.

```suggestion
    "Kitware",
```

Reply:
🙇‍♂